apps/portal/src/components/Global.styles.js

```diff
@@ -38,7 +38,7 @@ export const GlobalStyles = `
 
     body {
         margin: 0px;
```

Comment:
Note for whoever will review this PR.
When testing signin popup, I noticed some font and paragraph changes on the page: headings font looked smaller, and there was a bigger separation between paragraphs (only when popup is open). This was caused by styles injected in the popup which were previously isolated in the iframe and now are embed in the body. I fixed the visual glitch by adding Inter font and removing parragraph bottom margin from global styles, although I am not 100% sure this was the correct approach.